controllers/volumereplication_controller.go

```diff
@@ -200,19 +202,21 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		requeueForResync, replicationErr = r.resyncVolume(instance, volumeHandle, parameters, secret)
 
 	default:
-		replicationErr = fmt.Errorf("unsupported image state %q", instance.Spec.ReplicationState)
-		r.Log.Error(replicationErr, "unsupported image state", "Name", instance.Name, "ReplicationState", instance.Spec.ReplicationState)
+		replicationErr = fmt.Errorf("unsupported image state")
+		r.Log.Error(replicationErr, "given image state is not supported", "ReplicationState", instance.Spec.ReplicationState)
```

Comment:
the image needs to be changed to volume.

Reply:
Done